src/main/app/fact/FactApi.ts

```diff
@@ -162,7 +162,7 @@ export class FactApi {
 
   public getContactTypes(): Promise<ContactType[]> {
     return this.axios
-      .get(`${this.adminBaseUrl}/contactTypes`)
+      .get(`${this.adminUrl}/contactTypes`)
```

Comment:
Just double checking, resolve if so: 

url is changed to /admin/contactTypes on the api as well? 

As opposed to /admin/courts/contactTypes ? 

Is the latter still used by anything at all?

Reply:
later is not being used now will create RP to remove in next task